docs/configuration.md

```diff
@@ -364,6 +376,18 @@ Permitted values are `error`, `warn`, `info` and `debug`. The default is
         region: fr
         container: containername
         rootdirectory: /swift/object/name/prefix
+      oss:
```

Comment:
Including the entire file and its options is pretty standard.  It lets readers compare and contrast elements and features.  If we didn't do this, we force multiple clicks to discover.